### ISSUE TITLE
remixd auto exit when file hardhat.config.ts is exist

### ISSUE THREAD

**Issue by gzliudan** [ethereum/remix-project#3196]
If file hardhat.config.ts is exist, remixd 0.6.8 would auto exit when remix is connected to localhost. And remix would report below error when click any file or sub folder:
```text
Connection to remixd terminated. Please make sure remixd is still running in the background.
```

How to reproduce:
```shell
git clone https://github.com/safe-global/safe-contracts.git
cd safe-contracts
remixd
```
Then open https://remix.ethereum.org/ in chrome and connect to localhost.

**Comment by gzliudan** [comment]
This is my test vide:

https://user-images.githubusercontent.com/7695325/206645482-9c9e8934-a8d3-4426-9635-58c24ed848bb.mp4

Test environment: 
- Ubuntu: 22.04.01
- Node: v16.18.1
- Chrome: Version 108.0.5359.98 (Official Build) (64-bit)

Also has same problem on windows 10 with enviroment:
- Node: v16.18.0
- Edge: 108.0.1462.42

**Comment by Aniket-Engg** [comment]
@gzliudan  Can you check if it is fixed with remixd v0.6.10 ?

**Comment by gzliudan** [comment]
Ok, I will test new version and tell you the result tomorrow.

**Comment by gzliudan** [comment]
Yes, the problem is fixed in remixd v0.6.10.
